utils/cliutils/commandsflags.go

```diff
@@ -1141,26 +1173,26 @@ var flagsMap = map[string]cli.Flag{
 		Name:  vars,
 		Usage: "[Optional] List of variables in the form of \"key1=value1;key2=value2;...\" to be replaced in the template. In the template, the variables should be used as follows: ${key1}.` `",
 	},
-	groups: cli.StringFlag{
-		Name: groups,
+	rtAtcGroups: cli.StringFlag{
+		Name: Groups,
 		Usage: "[Default: *] A list of comma-separated groups for the access token to be associated with. " +
 			"Specify * to indicate that this is a 'user-scoped token', i.e., the token provides the same access privileges that the current subject has, and is therefore evaluated dynamically. " +
```

Comment:
```suggestion
			"Specify * to indicate that this is a 'user-scoped token', i.e., the token provides the same access privileges that the current subject has, and is therefore evaluated dynamically.` `" +
```

Reply:
there's another line to the usage and it has the suffix